okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -219,6 +219,10 @@ private ConnectionSpec supportedSpec(SSLSocket sslSocket) {
     private String[] tlsVersions;
     private boolean supportsTlsExtensions;
 
+    public Builder() {
```

Comment:
Drop this please! I want new instances to have to choose whether they're derived from `MODERN_TLS` or `COMPATIBLE_TLS`.